packages/core/src/main/start-main-application/lens-window/application-window/create-electron-window.injectable.ts

```diff
@@ -119,14 +110,7 @@ const createElectronWindowInjectable = getInjectable({
 
       applicationWindowState.manage(browserWindow);
 
-      browserWindow.webContents.session.setCertificateVerifyProc((request, shouldBeTrusted) => {
-        const { certificate } = request;
-        const cert = new X509Certificate(certificate.data);
-        const shouldTrustCert = cert.raw.length === lensProxyX509Cert.raw.length
-          && timingSafeEqual(cert.raw, lensProxyX509Cert.raw);
-
-        shouldBeTrusted(shouldTrustCert ? ChromiumNetError.SUCCESS : ChromiumNetError.RESULT_FROM_CHROMIUM);
-      });
+      browserWindow.webContents.session.setCertificateVerifyProc(sessionCertificateVerifier);
```

Comment:
Responsibility extracted, nice :)

I spent a moment thinking if this responsibility of "certificate verification" should be abstracted to eg. `createLensWindowInjectable`, and to have no knowledge of Electron. But the implementation seems to be very Electron-specific, making this the correct place.